configure.ac

```diff
@@ -2718,8 +2718,8 @@ AS_IF([test "x$with_cppunit" != "xno"],[
     AS_UNSET(squid_cv_cppunit_version)
 
     SQUID_STATE_SAVE(squid_cppunit_state)
-    AS_VAR_APPEND(CXXFLAGS,[$LIBCPPUNIT_CFLAGS])
-    AS_VAR_APPEND(LIBS,[$LIBCPPUNIT_LIBS])
+    CXXFLAGS="${CXXFLAGS}${CXXFLAGS+ }${LIBCPPUNIT_CFLAGS}"
+    LIBS="${LIBS}${LIBS+ }${LIBCPPUNIT_LIBS}"
```

Comment:
> Personally, I would be uncomfortable adding an advanced-looking ${NAME+ }
According to several documents online (e.g. https://en.wikibooks.org/wiki/Bourne_Shell_Scripting/Variable_Expansion#Substitution_with_a_default_value , http://www.grymoire.com/Unix/Sh.html#uh-33 ) , this is a standard construct in Bourne shell.

The trick you suggested was my first attempt, it didn't really work out well, that's why I decided to rework the callers, since it's only 2. The main objective of AS_VAR_APPEND is performance by using += when available, we don't really care about it here IMO.

> eval $[]1=\$$[]1\$[]2

It seems like they're using tricks to delay expansion and evaluation; the latter maybe also helps support variable expansion while doing it.

At this time I'd live as-is (even the whitespace won't hurt); if we want to get more performance we may investigate adding AS_VAR_APPEND extensively as its own project.